deploy/olm-catalog/service-assurance-operator/0.1.1/service-assurance-operator.v0.1.1.clusterserviceversion.yaml

```diff
@@ -110,11 +111,6 @@ spec:
       kind: Certificate
       name: certificates.certmanager.k8s.io
       version: v1alpha1
-    - description: Creation of an ElasticSearch cluster
```

Comment:
Removed the requirement to have the API provided by ECL. Did not replace with the ECK API since that will need to be pre-installed for now. We don't want to force it to be a requirement since it might not be available to us.